deepchecks/vision/utils/transformations.py

```diff
@@ -118,24 +147,71 @@ def get_robustness_augmentations(cls, data_dim: Optional[int] = 3):
         return augmentations
 
 
-def get_transforms_handler(transforms):
+class TorchTransformations(AbstractTransformations):
+    """Class containing supporting functions for torch transforms."""
+
+    @classmethod
+    def add_augmentation_in_start(cls, aug, transforms: T.Compose):
+        """Add given transformations to the start of given transforms object."""
+        if isinstance(aug, (albumentations.Compose, albumentations.BasicTransform)):
+            alb_aug = aug
+
+            class TorchWrapper:
+                def __call__(self, image):
+                    if isinstance(image, Image.Image):
+                        return T.ToPILImage()(alb_aug(image=np.array(image))['image'])
+                    elif isinstance(image, torch.Tensor):
+                        image = image.cpu().detach().numpy()
+                        return T.ToTensor()(alb_aug(image=image)['image'])
+                    return alb_aug(image=image)['image']
+            aug = TorchWrapper()
+        elif not isinstance(aug, torch.nn.Module):
+            raise DeepchecksValueError(f'Transforms is of type torch, can\'t add to it type '
```

Comment:
this is a bit weird here, no? can't add torch transform inside TorchTransformations

Reply:
it's if not